docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst

```diff
@@ -366,14 +406,10 @@ Note that when using variables from tasks in this way we use double quotes (``"`
 Troubleshooting
 ===============
 
-If you receive an connection error please double check the inventory and Playbook for typos or missing lines. If the issue still occurs follow the debug steps in :doc:`network_debug_troubleshooting`.
+If you receive an connection error please double check the inventory and Playbook for typos or missing lines. If the issue still occurs follow the debug steps in :ref:`network_debug_troubleshooting`.
```

Comment:
Playbook should be lowercase.

Reply:
yeah agreed, unless we have Ansible Playbook, it should just be playbook (lowercase)